index.html

```diff
@@ -8,135 +8,172 @@
     <link rel="stylesheet" href="styles/index.css" />
   </head>
 
-  <body class="app-grid">
-    <header class="app-grid__header">
+  <body>
+    <header class="header">
       <h1>Quiz App</h1>
     </header>
 
-    <main class="app-grid__main">
-      <section class="app-grid__main__card">
+    <main class="app-grid">
+      <section class="app-grid__card">
+        <img
+          src="img/icons/bookmark-regular.svg"
+          alt="bookmark"
+          class="app-grid__card--icon"
+        />
         <h2>What's your name?</h2>
-        <details class="app-grid__main__card__show-answer">
-          <summary class="app-grid__main__card__show-answer--toggle-show">
+        <details class="app-grid__card__show-answer">
+          <summary class="app-grid__card__show-answer--toggle-show">
             Show Answer
           </summary>
-          <p class="app-grid__main__card__show-answer__answer">Mandy</p>
-          <ul class="app-grid__main__card__tags">
-            <li class="app-grid__main__card__tags--tag">category</li>
+          <p class="app-grid__card__show-answer__answer">Mandy</p>
+          <ul class="app-grid__card__tags">
```

Comment:
Here you have each list item in a separate `ul`-element. It would make more sense to have one list with many list items. The usage of BEM is also not entirely correct, again with the modifier.
```suggestion
          <ul class="card__tag-list">
            <li class="card__tag">tag 1</li>
            <li class="card__tag">tag 2</li>
            <li class="card__tag">tag 3</li>
            <li class="card__tag">tag 4</li>
          </ul>
```